EXAMPLE_README.md

```diff
@@ -141,19 +133,22 @@ In order for the app to function correctly, the user must set up their own envir
 # 4️⃣ Testing
 
 🚫Document what you used for testing and why
+- SuperTest
+- Jest
+- Cypress
 
 # 4️⃣ Installation Instructions
 
 🚫explain how to install the required dependencies to get this project up and running with yarn and NPM
+- cd into the main root folder and then type 'npm i'
+- then type 'npm start'
 
 ## Other Scripts
 
 🚫replace these examples with your own
 
-    * typecheck - runs the TypeScript compiler
-    * build - creates a build of the application
-    * start - starts the production server after a build is created
-    * test - runs tests in **tests** directory \* eject - copy the configuration files and dependencies into the project so you have full control over them
+    * npm start - starts the production server after a build is created
+    * start: "craco --max_old_space_size=4096 start" - Increases JavaScript heap size
```

Comment:
Keep the formatting the same for each bullet point.
Check package.json for other available scripts and add them here.